pyengine/katana.py

```diff
@@ -58,10 +75,26 @@ def matcher(qty,book):
 
     return slices
 
-def quotetrimmer(limitprice,book):
+def quotetrimmer(side,limitprice,book):
+    if side == 'buy':
```

Comment:
ideally can streamline the buy vs sell thing so you can use the same function for both quotetrimmer and matcher rather than write this out twice.